pkg/models/timestamp_test.go

```diff
@@ -37,7 +37,7 @@ func TestTimestamp(t *testing.T) {
 		{"rfc3339", "2021-11-04T01:02:03Z", "2021-11-04T01:02:03Z"},
 		{"date", "2021-04-05", "2021-04-05T00:00:00Z"},
 		{"datetime", "2021-04-05 14:45:36", "2021-04-05T14:45:36Z"},
-		{"datetime-tz", "2021-04-05 14:45:36 PDT", "2021-04-05T14:45:36Z"},
+		{"datetime-tz", "2021-04-05 14:45:36 PDT", "2021-04-05T14:45:36-07:00"},
```

Comment:
This test was strangely failing for me, but I assume must have worked for previous PRs. At least my change also seems to pass the Github build.

Reply:
Ok, it fails on Github, will revert.